features/Steps/MailSteps.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Steps;
+
+use function PHPUnit\Framework\assertEquals;
+
+trait MailSteps
+{
+    /**
+     * @When I delete all mails
+     */
+    public function iDeleteAllMails(): void
+    {
+        $this->getMailPitClient()->delete([]);
+    }
+
+    /**
+     * When I get the latest mail
+     */
+    public function iGetTheLatestMail(): void
+    {
+        $this->getMailPitClient()->get();
+    }
+
+    /**
+     * @When a mail has been sent from :from to :to with :subject and :messageType
+     */
+    public function aMailHasBeenSentFromToWithAnd(string $from, string $to, string $subject, string $messageType): void
+    {
+        $mailobject = $this->getMailPitClient()->get();
```

Comment:
This is not clear that this is the last mail, so I would create two methods, `getLatestEmail` and `getEmailById`. But for now we only need `getLatestEmail`